packages/prosemirror-lwdita-demo/cypress/e2e/welcome-note.cy.ts

```diff
@@ -0,0 +1,78 @@
+/*!
+Copyright (C) 2020 Evolved Binary
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as
+published by the Free Software Foundation, either version 3 of the
+License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+function clearLocalStorage() {
+  window.localStorage.clear();
+}
+
+describe('The Welcome note:', () => {
+  let loadPage: Cypress.Chainable<Cypress.AUTWindow>,
+      note: { should: (arg0: string) => void; find: (arg0: any) => any; },
```

Comment:
This type is extremely ambiguous can you add a comment after it for the docs.

Reply:
@marmoure Done, noted in internal documentation.

Reply:
>noted in internal documentation.
And where is that?, what about the code itself?

Reply:
Fixed with 18d7409